scripts/lib/CIME/build.py

```diff
@@ -112,9 +112,206 @@ def _build_model(build_threaded, exeroot, clm_config_opts, incroot, complist,
     return logs
 
 ###############################################################################
-def _build_checks(case, build_threaded, comp_interface, use_esmf_lib,
-                  debug, compiler, mpilib, complist, ninst_build, smp_value,
-                  model_only):
+def post_build(case, logs):
+###############################################################################
```

Comment:
This doesn't look right. It's showing lots of code added that should have already been there.